content/en/cloud_cost_management/aws.md

```diff
@@ -135,20 +135,111 @@ Attach the new S3 policy to the Datadog integration role.
 
 **Note:** Data can take up to 48 to 72 hours after setup to stabilize in Datadog.
 
-### Cost types
+## Cost types
 
-You can visualize your ingested data using the following cost types:
+Visualize your ingested data using out-of-the-box cost types. Cloud cost metric names contain various terms that describe how they are calculated.
 
-| Cost Type            | Description           |
+### On-demand
+**On-demand** costs are the costs of usage at the public, on-demand rate published by AWS. This excludes all savings plans, reservations, discounts, taxes, and fees.
+
+These costs do not provide an accurate representation of what is paid to AWS, so it's not recommended to use this metric in most cases.
+
+### Amortized vs. unblended costs
+**Amortized** cost metrics distribute some pre-payments (like up-front savings plan fees) across the entire discount term. This is also called _accrual basis_. As savings plans are applied to covered usage, the costs for that savings plan appear as usage charges. The unused portion appears as a fee.
```

Comment:
```suggestion
**Amortized** cost metrics distribute some pre-payments (such as up-front savings plan fees) across the entire discount term. Another name for amortized cost metrics is _accrual basis_. As savings plans are applied to covered usage, the costs for that savings plan appear as usage charges. The unused portion appears as a fee.
```

Reply:
Not sure about this one, since we actually don't call this anywhere in our own product... it's more like "this type of cost reporting is also known as accrual basis / cash basis".

Reply:
Got it. Thanks for explaining.